dbms/src/Storages/Page/V3/Universal/tests/gtest_checkpoint.cpp

```diff
@@ -587,4 +637,142 @@ try
 }
 CATCH
 
+class UniversalPageStorageServiceCheckpointTest : public DB::base::TiFlashStorageTestBasic
+{
+public:
+    void SetUp() override
+    {
+        TiFlashStorageTestBasic::SetUp();
+        auto path = getTemporaryPath();
+        dropDataOnDisk(path);
+        createIfNotExist(path);
+        auto delegator = std::make_shared<DB::tests::MockDiskDelegatorSingle>(path);
+        auto & global_context = DB::tests::TiFlashTestEnv::getGlobalContext();
+        uni_ps_service = UniversalPageStorageService::create(global_context, "test.t", delegator, std::move(path), PageStorageConfig{.blob_heavy_gc_valid_rate = 1.0});
+        s3_client = S3::ClientFactory::instance().sharedClient();
+        bucket = S3::ClientFactory::instance().bucket();
```

Comment:
Just curious, is this running in a mocked environment, or it will create an actual bucket?